lambda_functions/va_profile/va_profile_opt_in_out_lambda.py

```diff
@@ -228,7 +226,7 @@ def va_profile_opt_in_out_lambda_handler(  # noqa: C901
     logger.info('POST request received.')
     logger.debug('POST event: %s', event)
 
-    global integration_testing_public_cert
+    global va_profile_public_cert, integration_testing_public_cert
```

Comment:
Nit. I believe this is only necessary if you want to assign this global variable to something else within this scope. Not something that needs to be change.